### ISSUE TITLE
JEI transfer handler causing issues on Neoforge

### ISSUE THREAD

**Issue by thiakil** [emilyploszaj/emi#468]
```
java.lang.ClassCastException: class net.minecraft.world.item.crafting.ShapedRecipe cannot be cast to class net.minecraft.world.item.crafting.RecipeHolder (net.minecraft.world.item.crafting.ShapedRecipe and net.minecraft.world.item.crafting.RecipeHolder are in module minecraft@1.20.4 of loader 'TRANSFORMER' @3595086b)
    at TRANSFORMER/mekanism@10.5.7/mekanism.client.jei.FormulaicRecipeTransferInfo.canHandle(FormulaicRecipeTransferInfo.java:18)
    at TRANSFORMER/jei@17.3.0.49/mezz.jei.library.transfer.BasicRecipeTransferHandler.transferRecipe(BasicRecipeTransferHandler.java:79)
    at TRANSFORMER/emi@1.1.2-SNAPSHOT+1.20.4+neoforge/dev.emi.emi.jemi.JemiRecipeHandler.jeiCraft(JemiRecipeHandler.java:134)
    at TRANSFORMER/emi@1.1.2-SNAPSHOT+1.20.4+neoforge/dev.emi.emi.jemi.JemiRecipeHandler.render(JemiRecipeHandler.java:90)
    at TRANSFORMER/emi@1.1.2-SNAPSHOT+1.20.4+neoforge/dev.emi.emi.screen.RecipeScreen.render(RecipeScreen.java:238)
    at TRANSFORMER/minecraft@1.20.4/net.minecraft.client.gui.screens.Screen.renderWithTooltip(Screen.java:106)
    at TRANSFORMER/neoforge@20.4.198/net.neoforged.neoforge.client.ClientHooks.drawScreenInternal(ClientHooks.java:399)
    at TRANSFORMER/neoforge@20.4.198/net.neoforged.neoforge.client.ClientHooks.drawScreen(ClientHooks.java:393)
    at TRANSFORMER/minecraft@1.20.4/net.minecraft.client.renderer.GameRenderer.render(GameRenderer.java:1137)
    at TRANSFORMER/minecraft@1.20.4/net.minecraft.client.Minecraft.runTick(Minecraft.java:1263)
    at TRANSFORMER/minecraft@1.20.4/net.minecraft.client.Minecraft.run(Minecraft.java:818)
    at TRANSFORMER/minecraft@1.20.4/net.minecraft.client.main.Main.main(Main.java:244)
```

According to the JEI API in my sources, `RecipeTypes.CRAFTING` is supposed to be `RecipeHolder`

EMI: `emi-1.1.3+1.20.4+neoforge`

**Comment by emilyploszaj** [comment]
I believe I released EMI based on an older version of JEI before it was released on Neo, which would be causing these errors and should be resolved.

**Comment by emilyploszaj** [comment]
Should be fixed in EMI 1.1.4
